test/se/vidstige/jadb/test/unit/MockedTestCases.java

```diff
@@ -99,7 +100,7 @@ public void testGetProps() throws Exception {
         server.add("serial-123");
         server.expectShell("serial-123", "getprop").returns("[] = nope\nx\n(");
         JadbDevice device = connection.getDevices().get(0);
-        Map<String, String> x = device.getprop();
+        Map<String, String> x = new PropertyManager(device).getprop();
```

Comment:
I just put `x` here to mean a placeholder. Let's rename to `properties` or such?